code/modules/admin/admin.dm

```diff
@@ -849,6 +849,17 @@ var/global/floorIsLava = 0
 	world.update_status()
 	feedback_add_details("admin_verb","TR") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/datum/admins/proc/toggleatime()
+	set category = "Server"
+	set desc="Sets the respawn time"
+	set name="Set Respawn Timer"
+	var/message = input("Enter time in minutes:") as message
+	respawntime = text2num(message)
```

Comment:
this should be an arg to the proc, not a var.

This way the client is handles showing them the prompt when they click, rather than requiring a round trip to the server.

```dm
/datum/admins/proc/toggleatime(respawntime as num)
```
Byond will handle sanitizing it as a number.